.github/workflows/master.yml

```diff
@@ -254,6 +254,28 @@ jobs:
           export MLFLOW_HOME=$(pwd)
           pytest tests/models
 
+  evaluate:
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          repository: ${{ github.event.inputs.repository }}
+          ref: ${{ github.event.inputs.ref }}
+      - uses: ./.github/actions/setup-python
+      - uses: ./.github/actions/setup-pyenv
+      - uses: ./.github/actions/setup-java
+      - name: Install dependencies
+        run: |
+          source ./dev/install-common-deps.sh
+          pip install pyspark "numpy<1.24.0"
```

Comment:
Can we leave a comment explaining why we have this pin and file an internal ticket for next sprint to track removing the pin? Feel free to merge after that. Thanks @BenWilson2 !

Reply:
Ticket's in :D (earlier today, just in case ;) ) Adding comment now